src/controllers/employerLogin.ts

```diff
@@ -0,0 +1,21 @@
+import { Request, Response, NextFunction } from 'express'
+import passport from 'passport'
+
+import { NotFoundError, UnauthorizedError } from '../helpers/apiError'
+//import Employer from '../entities/Employer.postgres'
+
+export const localLogin = async (
+  req: Request,
+  res: Response,
+  next: NextFunction
+) => {
+  try {
+    await passport.authenticate('local', function (error, employer) {
+      if (!employer) {
+        next(new NotFoundError('Email or password not found', error))
+      }
+    })
```

Comment:
what about the case where `employer` exists ?

Reply:
success